### PR TITLE
Enable v0.1 of estimated data on the map (under feature-flag) 🗺️ 

### PR DESCRIPTION
**Pull request by madsnedergaard**
### Description

_Part of #3575_

Enables using estimated data on the map and shows a message in the panel when data is estimated ⚡  

#### Try it out

Add a `?feature=estimations` parameter to the URL once this has been deployed: https://app.electricitymap.org/zone/IS?feature=estimations

#### Preview

**USA BEFORE**

Screenshot 2021-12-07 at 14 58 56

**USA AFTER**

Screenshot 2021-12-07 at 14 58 28

**PANEL**

Screenshot 2021-12-07 at 15 22 37

### CODE REVIEW

File: web/locales/en.json [hunk 1]
```diff
@@ -42,6 +42,9 @@
         "noDataAtTimestamp": "Data is temporarily unavailable for the selected time",
         "noLiveData": "Live data is temporarily unavailable for this area",
         "dataIsDelayed": "Data for this region can be up to %s hours delayed. Live data cannot be displayed.",
+        "dataIsEstimated": "Data for this region is delayed. Real-time values are estimated. Read more about our estimation methodology <a href=\"https://github.com/electricityMap/electricitymap-contrib/wiki/Estimation-methods\" target=\"_blank\">here</a>.",
+        "estimatedDisclaimer": "The real-time data is estimated based on the data from the source and the assumptions of the model. The data is not guaranteed to be accurate.",
+        "helpUs": "Help us identify the problem by taking a look at the <a href=\"%s\" target=\"_blank\">runtime logs</a> or contact the data provider.",
```
Comment (skovhus) [inline web/locales/en.json:47]: Source -> data provider

File: web/locales/en.json [hunk 2]
```diff
@@ -42,6 +42,9 @@
         "noDataAtTimestamp": "Data is temporarily unavailable for the selected time",
         "noLiveData": "Live data is temporarily unavailable for this area",
         "dataIsDelayed": "Data for this region can be up to %s hours delayed. Live data cannot be displayed.",
+        "dataIsEstimated": "Data for this region is delayed. Real-time values are estimated. Read more about our estimation methodology <a href=\"https://github.com/electricityMap/electricitymap-contrib/wiki/Estimation-methods\" target=\"_blank\">here</a>.",
```
Comment (skovhus) [inline web/locales/en.json:45]: Let us remove actual link in this message and pass it in using %s

File: web/src/sagas/index.js [hunk 3]
```diff
@@ -33,13 +33,19 @@ function* fetchZoneHistory(action) {
 
 function* fetchGridData(action) {
   const { features } = action.payload || {};
-  let queryParamString = '?preview=1';
+  let endpoint = '/v3/state?preview=1';
```
Comment (skovhus) [inline web/src/sagas/index.js:36]: Let us get rid of the “preview” leftover which isn’t used.

**Review by Kongkille — APPROVED** [review]

File: web/locales/en.json (hunk 1 above)
Comment (skovhus) [reply web/locales/en.json:47]: @Kongkille any strong feeling here?

**Review by skovhus — APPROVED** [review]

**Comment by skovhus** [comment]
The suggestions in this PR can be fixed after getting this out. :)

File: web/locales/en.json (hunk 1 above)
Comment (Kongkille) [reply web/locales/en.json:47]: I think this is fine for now :) 

@livmontmerle any thoughts on the copy for the feature?

**Pull request merged**